libs/ledger-live-common/src/market/api/api.ts

```diff
@@ -137,11 +140,25 @@ async function listPaginated({
     }&price_change_percentage=${range}` +
     `${ids.length > 0 ? `&page=1&&ids=${ids.toString()}` : `&page=${page}`}`;
 
-  const { data } = await network({
+  let { data } = await network({
     method: "GET",
     url,
   });
 
+  if (top100) {
+    data = data
+      .sort(
+        (a, b) =>
+          b[`price_change_percentage_${range}_in_currency`] -
+          a[`price_change_percentage_${range}_in_currency`]
+      )
+      .filter((currency) => {
+        if (!search) return true;
+        const match = `${currency.symbol}|${currency.name}`;
+        return match.toLowerCase().includes(search.toLowerCase());
+      });
```

Comment:
I think moving the filter BEFORE the sort helps the performance.

Reply:
Thank you for your feedback. I made the change :)